doc/modules/ROOT/pages/generate-jwt.adoc

```diff
@@ -140,8 +140,8 @@ Smallrye JWT supports the following properties which can be used to customize th
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
-|smallrye.jwt.encrypt.key-location|none|Location of a key which will be used to encrypt the claims or inner JWT when a no-argument encrypt() method is called.
-|smallrye.jwt.sign.key-location|none|Location of a key which will be used to sign the claims when either a no-argument sign() or innerSign() method is called.
+|smallrye.jwt.encrypt.key.location|none|Location of a key which will be used to encrypt the claims or inner JWT when a no-argument encrypt() method is called.
```

Comment:
I'm wondering if we should keep the deprecated key here? But with a mark that is deprecated?

Reply:
@radcortez Hi Roberto, since it is not related to MP JWT at the moment, I thought of following a somewhat aggressive deprecation policy. So the old property stays in 2.4.x, but will be gone in 3.0.0 - otherwise we'd have to carry it around for a long time. That said, these are 2.4.3 docs which are being changed in this PR so yes, good point, let me add this note here as well (I've been thinking only about updating the JavaDocs)

Reply:
@radcortez I've updated the docs - noted that the 2 other properties are deprecated and will be removed - but I'll also 'lost' your approval with this update - please suggest some further doc updates if you'd prefer, otherwise I'll merge once it is green